linode/helper/framework_import.go

```diff
@@ -36,3 +38,74 @@ func ImportStatePassthroughInt64ID(
 
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, attrPath, intID)...)
 }
+
+type IDTypeConverter func(values ...string) ([]any, diag.Diagnostics)
+
+func AllInt64(values ...string) ([]any, diag.Diagnostics) {
```

Comment:
Just to make them a bit more concise, maybe it'd make sense to name the type converters something like `BulkStringToInt64`?